angr/knowledge_plugins/variables/variable_manager.py

```diff
@@ -717,7 +718,7 @@ def get_variables(self, sort: Literal["stack"], collapse_same_ident: bool = Fals
     def get_variables(self, sort: Literal["reg"], collapse_same_ident: bool = False) -> list[SimRegisterVariable]: ...
     @overload
     def get_variables(
-        self, sort: None, collapse_same_ident: bool = False
+        self, sort: None = None, collapse_same_ident: bool = False
```

Comment:
Does sort have a type besides None?

Reply:
Yes, look at the two overload cases above.

Reply:
oh, my bad